scout/adapter/mongo/case.py

```diff
@@ -521,7 +522,7 @@ def rna_cases(self, owner):
         Returns:
             list of case _ids
         """
-        EXISTS_NOT_NULL = {"$exists": True, "$nin": [None, ""]}
+        EXISTS_NOT_NULL = {EXISTS: True, "$nin": [None, ""]}
```

Comment:
I dunno, this looks like pleasing some silly code evaluation? It's not like mongo are going to change that function anytime soon?! And if so we could search-replace it just as easy. The rule is for different situations, like natural language to be translated, or more complex ones that could go wrong. "EXISTS_NOT_NULL" is good, and e.g. `EXISTS = {"$exists": True}` or whatnot would be ok in my book!

Reply:
Yes, of course it's pleasing that code checker, I would never think of such replacement myself. I like your suggestion but if you check in the code you'll find that the "exists" is used in slightly different situations, for instance:

- set_value={EXISTS: False}
- set_value={EXISTS: True, "$ne": []} # there are a bunch of these
- set_value={EXISTS: True, "$eq": True},
- EXISTS_NOT_NULL = {EXISTS: True, "$nin": [None, ""]}

That's why I decided to replace only "$exists" and not the whole dict. This way it is more easy to read and doesn't trigger the code checker. I'd like to leave it like this if it's not a problem for you

Reply:
I feel that if I replace the cases above with 5 different constants then it will be much less readable. If you don't like the EXISTS = "$exists" replacement I can simply revert to how the code was before and we ignore the code checker warning instead

Reply:
Whatever! It's just a constant string regardless, and the only reason to change should be to make it more readable and maintainable. Since there is a "EXISTS_NOT_NULL" already, adding the others as similar full constants would make sense to me, but its no big deal.